sechub-doc/src/docs/asciidoc/sechub-tutorials.adoc

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+// ====================================
+// SecHub user Tutorials
+// ====================================
+// enable tutorial content
+:tutorials:
+
+include::documents/gen/server-version.adoc[]
+
+include::documents/config.adoc[]
+
+= image:sechub-logo.png[link="{webPage}"] SecHub Tutorials
+
```

Comment:
I would expect a short and concise intro statement, explaining what the tutorials are for.